src/controller/user.controller.ts

```diff
@@ -0,0 +1,123 @@
+import { Request, Response } from "express";
+import { NOT_FOUND, INTERNAL_SERVER_ERROR, OK, CREATED ,SocialMediaType, BAD_REQUEST } from "../utils/constants.utils";
+import userService from "../services/user.service";
+import { ISocialMediaURLS, IUser } from "../models/interfaces.models";
+import { ICreatorWorkMetadata } from "../models/interfaces.models";
+import { forEachChild } from "typescript";
+
+
+class UserController {
+
+    // get all the users
+    getCreatedUser = async (request: Request, response: Response) => {
+        try {
+            const users = await userService.getUser();
+            if (users) {
+                return response.status(OK).send(users);
+            }
+            else {
+                return response.status(NOT_FOUND).send({
+                    message: `User not found`
+                });
+            }
+        } catch (error: any) {
+            return response.status(NOT_FOUND).send({
```

Comment:
This should be an Internal Server Error because we are not trying to get a single resource and it will be coming from MongoDB